plugins/woocommerce/src/StoreApi/Utilities/QuantityLimits.php

```diff
@@ -92,51 +126,32 @@ public function limit_to_multiple( int $number, int $multiple_of, string $roundi
 	/**
 	 * Check that a given quantity is valid according to any limits in place.
 	 *
-	 * @param integer           $quantity Quantity to validate.
-	 * @param \WC_Product|array $cart_item Cart item.
+	 * @param integer $quantity Quantity to validate.
+	 * @param array   $cart_item Cart item.
 	 * @return \WP_Error|true
 	 */
 	public function validate_cart_item_quantity( $quantity, $cart_item ) {
-		$limits = $this->get_cart_item_quantity_limits( $cart_item );
+		$limits  = $this->get_cart_item_quantity_limits( $cart_item );
+		$product = $cart_item['data'];
```

Comment:
Same here, is it possible for $cart_item['data'] not to be set? Should we check before accessing it?

Reply:
https://github.com/woocommerce/woocommerce/pull/52154/commits/aae25d6c0fdf1aafa2d1ee02fc90f99ff102c1b1